### PR TITLE
PHP7 consistent ordering for splitHttpAcceptHeader

### PR DESCRIPTION
**Pull request by mdeheij**
In PHP7 sorting order is changed, splitHttpAcceptHeader was
relying on unspecified behaviour of arsort(). Sorting order
is now made explicit.

### CODE REVIEW

File: test/lib/request/sfWebRequestTest.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<?php
+use PHPUnit\Framework\TestCase;
+
+require_once (__DIR__ . '/../../../lib/request/sfWebRequest.class.php');
+
+class sfWebRequestTest extends TestCase
```
Comment (nicoschoenmaker) [inline test/lib/request/sfWebRequestTest.php:6]: Missing `@covers`

File: phpunit.xml.dist [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit colors                 = "true"
+	 bootstrap              = "vendor/autoload.php"
```
Comment (linaori) [inline phpunit.xml.dist]: There is no .travis.yml file, so having tests without automating them is kinda pointless. You could add one with a build-matrix but you'll have to run all tests for that

Comment (nicoschoenmaker) [inline phpunit.xml.dist]: tab?

File: test/lib/request/sfWebRequestTest.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                    ' */*',
+                    ' text/xml',
+                    ' application/xml',
+                    'text/html',
+                    'text/javascript',
+                ]
+            ]
+        ];
+    }
+}
```
Comment (yannickl88) [inline test/lib/request/sfWebRequestTest.php]: Missing return at the end of the file

**Pull request merged**